dome.py

```diff
@@ -146,46 +146,40 @@
 #Not Tested 
 # option 1
 # def my_callback(channel):
-#	if GPIO.input(36) ==0):
+#	if GPIO.input(37) == 0):
 #		print("Falling edge")
 #		
-#	if GPIO.input(36) !=0):
+#	if GPIO.input(37) != 0):
 #		print("Rising edge")
-#	notch_ir = GPIO.input(36)
+#	notch_ir = GPIO.input(37)
 #	return notch_ir
 # then in clockwise and counterwise we have if notch_ir = 1 notches == notches + 1 
 # but I dont see how this is different from what we have now? this just seems the same? 
```

Comment:
The RISING/FALLING allows constant monitoring without being in a loop. This seems like it might work!